app/src/main/res/layout/fragment_suggested_edits_add_descriptions.xml

```diff
@@ -80,44 +80,79 @@
         <androidx.constraintlayout.widget.ConstraintLayout
             android:id="@+id/bottomButtonContainer"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:paddingBottom="8dp"
+            android:layout_height="50dp"
```

Comment:
The top of the `addDescriptionButton` has been cut off. The height of the container should be adjusted.

Reply:
Could you please give a screenshot @cooltey did this happen on a small screen?

Reply:
![Screenshot_1557181594](https://user-images.githubusercontent.com/2435576/57259201-90f25180-7013-11e9-95a6-0ef3a1995370.png)

It's on a large screen (1440 * 2880) emulator.

Reply:
Thank you... done.